app/views/pages/_contact.haml

```diff
@@ -10,7 +10,7 @@
     .row
       %ul.list-inline.text-center
         %li
-          %a.btn-social-contact.btn-contact-outline{:href => "https://twitter.com/BastienBarbe"}
+          %a.btn-social-contact.btn-contact-outline{:href => "https://twitter.com/intent/follow?screen_name=BastienBARBE"}
```

Comment:
Line is too long. [122/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace